### PR TITLE
Add Dether and LocalEthereum

### PR DESCRIPTION
**Pull request by john-light**
- Removed referral links 
- Made list alphabetical order
- Cleaned up some of the copy

### CODE REVIEW

**Comment by sassal** [comment]
Hey John - thanks for creating this PR.

The referral links belong to us (Eric and I) and the funds generated are used to reinvest back into EthHub!

We outlined our monetization strategy and principles in this blog post: https://medium.com/ethhub/building-a-sustainable-ethhub-4754c4c080a6.

**Comment by john-light** [comment]
@sassal ok thanks for letting me know! I'll revert those changes.

File: docs/using-ethereum/how-to-buy-ether.md [hunk 1]
```diff
@@ -1,23 +1,33 @@
...
+Binance is headquartered in Malta and constantly has some of the highest trading volume. Users can buy ETH with fiat or cryptocurrencies.
+
+[Buy ETH on Binance](https://www.binance.com)
 
 ## Coinbase
-Coinbase is based in San Fransisco and is one of the most reputable and well known exchanges in the world. Users can buy ETH with cash, credit or Bitcoin.
+Coinbase is based in San Fransisco and is one of the most reputable and well known exchanges in the world. Users can buy ETH with fiat or cryptocurrencies.
 
-[Buy ETH on Coinbase](https://www.coinbase.com/join/527bbccd0c46660a8a00003b)
+[Buy ETH on Coinbase](https://www.coinbase.com)
```
Comment (john-light) [inline docs/using-ethereum/how-to-buy-ether.md]:
```suggestion
[Buy ETH on Coinbase](https://www.coinbase.com/join/527bbccd0c46660a8a00003b)
```

File: docs/using-ethereum/how-to-buy-ether.md [hunk 2]
```diff
@@ -1,23 +1,33 @@
 # How to Buy Ether
 
-In order to [interact](https://docs.ethhub.io/using-ethereum/transactions/) with the Ethereum network, you will need to own ether (ETH). Unless you are mining, the only way to acquire ETH is from someone that already owns it. Below is a list of reputable online crypto exchanges where you can acquire ETH with fiat or other crypto. It is highly recommended to store the ETH you own in a [hardware wallet](https://docs.ethhub.io/using-ethereum/wallets/hardware/)!
+In order to [interact](https://docs.ethhub.io/using-ethereum/transactions/) with the Ethereum network, you will need to own ether (ETH). Unless you are mining, the only way to acquire ETH is from someone that already owns it. Below is a list of cryptocurrency exchanges where you can acquire ETH with fiat or cryptocurrencies. It is highly recommended to store the ETH you own in a [hardware wallet](https://docs.ethhub.io/using-ethereum/wallets/hardware/)!
+
+## Binance
+Binance is headquartered in Malta and constantly has some of the highest trading volume. Users can buy ETH with fiat or cryptocurrencies.
+
+[Buy ETH on Binance](https://www.binance.com)
```
Comment (john-light) [inline docs/using-ethereum/how-to-buy-ether.md]:
```suggestion
[Buy ETH on Binance](https://www.binance.com/?ref=10900939)
```

**Comment by sassal** [comment]
Awesome - thank you!

**Pull request merged**